### PR TITLE
LW-969 Initial infrastructure

### PR DESCRIPTION
**Pull request by mlkirkpatrick**
Just a single lambda with an API gateway, and a CodePipeline.

### CODE REVIEW

**Review by rdoughty — APPROVED** [review]

**Review by danthewolfe — APPROVED** [review]

File: lib/contentful-maps-pipeline-stack.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      new SlackApproval(this, 'SlackApproval', {
+        approvalTopic,
+        notifyStackName: props.slackNotifyStackName,
+      })
+    }
+
+    // TEST STAGE
+    pipeline.addStage({
+      stageName: 'DeployToTest',
+      actions: [deployToTestAction, manualApprovalAction],
```
Comment (jgondron) [inline lib/contentful-maps-pipeline-stack.ts:131]: Please add some form of automated tests such as smoke tests at a minimum

**Review by jgondron — CHANGES_REQUESTED** [review]

File: lib/contentful-maps-pipeline-stack.ts (hunk 1 above)
Comment (mlkirkpatrick) [reply lib/contentful-maps-pipeline-stack.ts]: Added test. Had to add a parameter (and appropriate permissions) to pass the api url down.

**Review by jgondron — APPROVED** [review]
